tests/test_search.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+from unittest.mock import patch
```

Comment:
This breaks for Python 2; [the mock object syntax is different](https://cpython-test-docs.readthedocs.io/en/latest/library/unittest.mock.html).